sefaria/views.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 import io
 import os
+from sefaria.sheets import get_sheet_categorization_info
```

Comment:
We want to follow this code style for ordering imports: https://www.python.org/dev/peps/pep-0008/#imports

For us this means generally a group of system imports, a group of django imports, then a group of sefaria imports.